duffel_api/utils.py

```diff
@@ -49,7 +49,21 @@ def maybe_parse_date_entries(key: str, value: Any) -> Union[str, datetime, date]
     return value
 
 
-def version():
+def identity(value: Any) -> Any:
+    return value
+
+
+def get_and_transform(dict: dict, key: str, fn=identity):
```

Comment:
@richcooper95: Does this seem like an idiomatic Python approach to this?

Reply:
It's certainly a valid way, I like it! I've suggested an alternative above, but I'm happy either way.

Is the `identity` function ever expected to be used? Isn't `get_and_transform(json, "foo")` == `json.get("foo")`?

I'd have thought we could just make `fn` a required arg of `Callable` type 😊

Reply:
Good question. I've refactored this now.